src/Microsoft.DotNet.Build.Tasks/PackageFiles/optional-tool-runtime/optional.json

```diff
@@ -2,6 +2,7 @@
   "frameworks": {
     "net45": {
       "dependencies": {
+        "Microsoft.DotNet.BinInspector": "1.0.0-alpha-00001",
```

Comment:
@dagood do we have the work item tracking this becoming PackageReferences instead of project.json?